src/main/java/uk/gov/pay/api/filter/LoggingFilter.java

```diff
@@ -29,8 +30,7 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
 
         String requestURL = ((HttpServletRequest) servletRequest).getRequestURI();
         String requestMethod = ((HttpServletRequest) servletRequest).getMethod();
-        Optional<String> requestIdMaybe = Optional.ofNullable(((HttpServletRequest) servletRequest).getHeader(HEADER_REQUEST_ID));
-        String requestId = requestIdMaybe.orElse("");
+        String requestId = StringUtils.defaultString(((HttpServletRequest)servletRequest).getHeader(HEADER_REQUEST_ID));
```

Comment:
now that you have made the change can you implement the same way at RestClientLoggingFilter:28